### PR TITLE
refactor: fund account tile

### PR DESCRIPTION
**Pull request by fbwoolf**
> Try out this version of the Hiro Wallet - download [extension builds](https://github.com/hirosystems/stacks-wallet-web/actions/runs/2468806178).

This PR finishes up the fund account tile on the FundPage so we can release. The provider link to Hiro faq sends params for use later.

Details found here: https://github.com/hirosystems/stacks-wallet-web/issues/2251#issuecomment-1149888899

Screen Shot 2022-06-08 at 2 31 20 PM

cc/ @kyranjamie @fbwoolf @beguene

### CODE REVIEW

File: src/app/query/hiro-config/hiro-config.query.ts
Comment (fbwoolf) [inline src/app/query/hiro-config/hiro-config.query.ts]: Change this back to `dev` before merging.

File: src/app/pages/fund/components/fiat-providers.utils.ts
Comment (beguene) [inline src/app/pages/fund/components/fiat-providers.utils.ts]: @fbwoolf Any particular reason why it's optional now ?

File: config/wallet-config.json
Comment (kyranjamie) [inline config/wallet-config.json]: Funky camelcase, and also can read as _has USA vailability_.

Avoid capitalising acronyms imo, recommend `hasUsaAvailability` (or spell the name fully)

File: src/app/pages/fund/components/fiat-provider-item.tsx
Comment (kyranjamie) [inline src/app/pages/fund/components/fiat-provider-item.tsx]: Same here, [Microsoft has guidelines for this](https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/141e06ef(v=vs.71)?redirectedfrom=MSDN), as does @zone117x also have opinions lol

Strictly speaking, as a two better acronym, this follows the rules, but imo it's still better to avoid two letter acronyms where possible. `USD` is a common abbreviation, so literally read this as _available inside usd..._.

Suggest: `availableInsideUsaDescription`

File: src/app/pages/fund/components/fiat-providers.utils.ts
Comment (kyranjamie) [reply src/app/pages/fund/components/fiat-providers.utils.ts]: Big +1

If this method is sometime invoked with undefined values, that's a concern of the code _invoking_ the function, not the function itself.

Comment (fbwoolf) [reply src/app/pages/fund/components/fiat-providers.utils.ts]: That is a mistake, thanks for catching. I was testing something and forgot to remove.

File: config/wallet-config.json
Comment (fbwoolf) [reply config/wallet-config.json]: Ok, thanks, yeah tried several things but didn't like any of them. Haha. Will change.

File: src/app/pages/fund/components/fiat-provider-item.tsx
Comment (fbwoolf) [reply src/app/pages/fund/components/fiat-provider-item.tsx]: I see that now, thx. Will change.

**Comment by fbwoolf** [comment]
Hoping to get an approval on this so can merge it today.

File: src/app/pages/fund/components/fiat-providers.utils.ts
Comment (kyranjamie) [inline src/app/pages/fund/components/fiat-providers.utils.ts]: This isn't a great method name imo. This fn makes a url for the faq page with provider and address. 

`makeFundAccountProviderFaqUrl` ? 🤔

**Review by kyranjamie — APPROVED** [review]

File: src/app/pages/fund/components/fiat-providers.utils.ts
Comment (fbwoolf) [reply src/app/pages/fund/components/fiat-providers.utils.ts]: Yeah, def better, I'll change.

**Pull request merged**
